force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -160,10 +193,10 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
-        Integer milliseconds;
-        return null; // Replace null with the variable you used to store the result
-    }    
+        Integer millisecondsPerMinute = 60000; // Make this value a constant
```

Comment:
Nice job avoiding the error message for the naming convention. It is actually happening though because the uppercase name is reserved for constants. What keyword can you add to the variable to make it a constant?